homeassistant/components/light/demo.py

```diff
@@ -23,17 +26,27 @@
 SUPPORT_DEMO = (SUPPORT_BRIGHTNESS | SUPPORT_COLOR_TEMP | SUPPORT_EFFECT |
                 SUPPORT_COLOR | SUPPORT_WHITE_VALUE)
 
+CONF_WITH_BASIC = 'demo_with_basic'
```

Comment:
Please don't add any config options to our demo platforms, especially to satisfy a test.